docker/hoot_external_command_nightly/scripts/RunCommandsAcrossContainers.sh

```diff
@@ -47,6 +47,17 @@ run-owt-import()
         "${OP_URL}" $OUTPUT_OSM
 }
 
+run-owt-import-bounds-file()
+{
+    hoot convert -D bounds.input.file=$BOUNDING_BOX -D overpass.api.query.path=$OQF -D schema.translation.script=$TRANSLATE_SCRIPT \
+        -D hoot.pkcs12.key.path=$CERT_PATH -D hoot.pkcs12.key.phrase=$CERT_PW -D overpass.api.host=owt.maxarmaps.com \
```

Comment:
I think you should replace hardcode `owt.maxarmaps.com` with another ENV VAR everywhere in this file